holmes/core/llm.py

```diff
@@ -36,6 +37,29 @@
 OVERRIDE_MAX_CONTENT_SIZE = environ_get_safe_int("OVERRIDE_MAX_CONTENT_SIZE")
 ROBUSTA_AI_MODEL_NAME = "Robusta"
 
+# ModelEntry represents a single LLM model configuration
+
+
+class ModelEntry(BaseModel):
+    model: str
+    # TODO: the name field seems to be redundant, can we remove it?
+    name: Optional[str] = None
```

Comment:
I did not find the name filed is used any place except in L493? and though it's passed to DefaultLLM, DefaultLLM does not actually use this parameters.  Can we remove it? @moshemorad 

https://github.com/robusta-dev/holmesgpt/blob/7a4fe4c4c4872ccf4ef64c85ca3c444c4fbe0fb8/holmes/config.py#L467-L469